extension/api.js

```diff
@@ -1,7 +1,11 @@
-const { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
```

Comment:
The `Services` import should not be removed either, as it was added in 88 - https://bugzilla.mozilla.org/show_bug.cgi?id=1698158 (https://hg.mozilla.org/mozilla-central/rev/7793d8d8a817).